### PR TITLE
fix: UAR-369: Fix-Page-Title-being-rendered-on-page

### PR DESCRIPTION
**Pull request by RobertBromfield**
### JIRA link

https://companieshouse.atlassian.net/browse/UAR-369

### Change description

The nunjucks block pageTitle seems not to work in a shared template so needed to be moved to individual pages.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/beneficial-owner-statements.html [hunk 1]
```diff
@@ -1,7 +1,3 @@
-{% block pageTitle %}
-  {% include "includes/page-title.html" %}
-{% endblock %}
-
 {% block backLink %}
```
Comment (AndreKramer7654) [inline views/includes/page-templates/beneficial-owner-statements.html:1]: move this out of share too please

File: views/beneficial-owner-statements.html [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
 
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
 {% block content %}
```
Comment (AndreKramer7654) [inline views/beneficial-owner-statements.html:13]: Don't we need to move the template content into here?

Comment (AndreKramer7654) [reply views/beneficial-owner-statements.html:13]: I guess we can choose not to but that was what registration had originally?

File: src/utils/navigation.ts [hunk 3]
```diff
@@ -96,7 +96,7 @@ export const NAVIGATION: Navigation = {
     nextPage: []
   },
   [config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL]: {
```
Comment (AndreKramer7654) [inline src/utils/navigation.ts:98]: any change to registration page for this? Probably it's already right but double checking that no changes needed for registration would be good.

File: src/utils/beneficial.owner.statements.ts [hunk 4]
```diff
@@ -10,14 +10,13 @@ import { saveAndContinue } from "../utils/save.and.continue";
 
 export const getBeneficialOwnerStatements = (req: Request, res: Response, next: NextFunction, registrationFlag: boolean) => {
   try {
-    const PAGE_NAME = registrationFlag ? config.BENEFICIAL_OWNER_STATEMENTS_PAGE : config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_PAGE;
     const BACK_LINK = registrationFlag ? config.ENTITY_URL : config.OVERSEAS_ENTITY_REVIEW_URL;
-    logger.debugRequest(req, `GET ${PAGE_NAME}`);
+    logger.debugRequest(req, `GET ${config.BENEFICIAL_OWNER_STATEMENTS_PAGE}`);
```
Comment (stephanie-ch) [inline src/utils/beneficial.owner.statements.ts:14]: use req.method approach for these log messages?

**Review by stephanie-ch — APPROVED** [review]

File: src/utils/navigation.ts (hunk 3 above)
Comment (bbell-companieshouse) [reply src/utils/navigation.ts:98]: Just checked and no changes have been made to registration. Both urls lead to the same view as expected

**Pull request merged**